src/engine.c

```diff
@@ -1,3 +1,10 @@
+int
+getColorComponents(uint32_t color, int *r, int *g, int *b) {
+  *r = color & 0xFF;
+  *g = (color & (0xFF << 8)) >> 8;
+  *b = (color & (0xFF << 16)) >> 16;
+}
```

Comment:
This method is expected to return an `int`, but does not do so, resulting in a warning.

In addition, please tag this method with `internal`.

Reply:
Both are the same issue 😆. I remember that I wanted `internal`, but accidentally typed `int`, and the the editor didn't warn about a missing return type 😆.